scripts/setup/register-server

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+EXTERNAL_HOST="zulip.example.com"
+ZULIP_ADMINISTRATOR="zulip-admin@example.com"
```

Comment:
How do these get set to the real values?

Try using `scripts/get-django-setting`.

Reply:
I used these value as demo just for manual testing. I dont found anything useful in ```scripts/get-django-setting```. These values are assigned in ```scripts/setup/generate_secrets.py```

Reply:
I used these value as demo just for manual testing. I dont found anything useful in ```scripts/get-django-setting```. These values are assigned in ```scripts/setup/generate_secrets.py```